### ISSUE TITLE
`man` does not work on latest PowerShell daily build

### ISSUE THREAD

**Issue by kborowinski** [PowerShell/PowerShell#19591]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
- [X] Search the existing issues.
- [X] Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md).
- [X] Refer to [Differences between Windows PowerShell 5.1 and PowerShell](https://docs.microsoft.com/powershell/scripting/whats-new/differences-from-windows-powershell).

### Steps to reproduce

1. Start new session on PowerShell daily build
2. Type `man` or `man Get-FileHash`
3. No help is displayed

Note: `Get-Help` works as expected

### Expected behavior

```console
Help should be displayed
```

### Actual behavior

```console
No help is displayed
```

### Error details

```console
No error is returned
```

### Environment data

```powershell
Name                           Value
----                           -----
PSVersion                      7.4.0-preview.3
PSEdition                      Core
GitCommitId                    7.4.0-preview.3-16-g2424ad83aa4d44fe9e8f507485744e00c66cde58
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Experimental features
```
PS C:\> Get-ExperimentalFeature | Select-Object Name, Enabled

Name                                          Enabled
----                                          -------
PSCommandNotFoundSuggestion                      True
PSCommandWithArgs                               False
PSCustomTableHeaderLabelDecoration               True
PSFeedbackProvider                               True
PSLoadAssemblyFromNativeCode                     True
PSModuleAutoLoadSkipOfflineFiles                 True
PSNativeCommandErrorActionPreference            False
PSNativeCommandPreserveBytePipe                 False
PSSubsystemPluginModel                           True
PSDesiredStateConfiguration.InvokeDscResource    True
```

### Visuals

![image](https://user-images.githubusercontent.com/16168755/235321011-ab217f0f-2076-4e09-9c41-1dde972be2af.png)

**Comment by doctordns** [comment]
`man` works in earlier versions, but in the latest BOTD, I can duplicate this.

Pretty sure this is not expected behaviour.

**Comment by SeeminglyScience** [comment]
Fixed by #19614

**Comment by SeeminglyScience** [comment]
(You can also probably just turn on the experimental feature fyi (`Get-ExperimentalFeature PSNativeCommandPreserveBytePipe | Enable-ExperimentalFeature`))
